QUEST_LV_0100.tsv

```diff
@@ -6249,7 +6249,7 @@ QUEST_LV_0100_20150317_006244	Cut Bramble's Roots.
 QUEST_LV_0100_20150317_006245	Approached the Bramble's roots but Gaigalas appeared. You must defeat Gaigalas before destroying Bramble's roots.
 QUEST_LV_0100_20150317_006246	Talk to Follower Jurga
 QUEST_LV_0100_20150317_006247	You cut off Bramble's roots. Return and talk to Follower Jurga.
-QUEST_LV_0100_20150317_006248	Defeat Gaigalas guarding Bramble's Root
+QUEST_LV_0100_20150317_006248	Defeat Gaigalas
```

Comment:
It's weird here because it shows up in a list alongside like:
"Defeat Gaigalas: 0/1
 Destroy Bramble's Root: 0/1"